packages/frontend/src/features/common/styles/constans/FontSize.tsx

```diff
@@ -1,17 +1,17 @@
 export enum FontSize {
-    paragraphDesktop = '18px',
-    paragraphMobile = '16px',
-    headerDesktop = '32px',
-    headerMobile = '30px',
-    subtitleDesktop = '22px',
-    subtitleMobile = '20px',
-    MEDIUM_LARGE = "28px",
-    VERY_SMALL="11px",
-    SMALL = "14px",
-    VERYLARGE = '23px',
-    MEDIUM = '18px',
-    MEDIUM_BIG = "23px",
-    NORMAL = "20px",
+    VERY_SMALL = "10px",
+    VERY_SMALLX = "11px",
+    VERY_SMALLXX = "12px",
+    XSMALL = "14px",
+    SMALL = "15px",
+    SMALLX = "16px",
+    XXMEDIUM = "18px",
```

Comment:
czemu X jest raz z prawej Very_smallx a raz z lewej XXMedium?

Reply:
założyłem że może to działać jak liczby rzymskie że jak x jest z tyłu to mamy 1 mniej a jak z przodu to jeden więcej